### PR TITLE
[Merlin Entertainments AU] New rulesets

### PR DESCRIPTION
**Pull request by youdly**
Related to https://github.com/EFForg/https-everywhere/pull/4002.

### CODE REVIEW

**Comment by J0WI** [comment]
Sorry for the delay, but I need more time to test all domains.

File: src/chrome/content/rules/Melbourne-Aquarium.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!--
+	For other Merlin Entertainments coverage, see Merlin-Entertainments.xml.
+-->
+<ruleset name="Melbourne Aquarium">
+	<target host="melbourneaquarium.com.au" />
+	<target host="m.melbourneaquarium.com.au" />
+	<target host="www.melbourneaquarium.com.au" />
```
Comment (J0WI) [inline src/chrome/content/rules/Melbourne-Aquarium.xml]: Mismatch: calendar.melbourneaquarium.com.au

File: src/chrome/content/rules/Sydney-Aquarium.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<!--
+	For other Merlin Entertainments coverage, see Merlin-Entertainments.xml.
+-->
+<ruleset name="Sydney Aquarium">
+	<target host="sydneyaquarium.com.au" />
+	<target host="m.sydneyaquarium.com.au" />
+	<target host="secure.sydneyaquarium.com.au" />
+	<target host="www.sydneyaquarium.com.au" />
```
Comment (J0WI) [inline src/chrome/content/rules/Sydney-Aquarium.xml]: Mismatch: calendar.sydneyaquarium.com.au

File: src/chrome/content/rules/Wildlifesydney.com.au.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<!--
+	For other Merlin Entertainments coverage, see Merlin-Entertainments.xml.
+-->
+<ruleset name="WILD LIFE Sydney Zoo">
+	<target host="wildlifesydney.com.au" />
+	<target host="m.wildlifesydney.com.au" />
+	<target host="secure.wildlifesydney.com.au" />
+	<target host="www.wildlifesydney.com.au" />
```
Comment (J0WI) [inline src/chrome/content/rules/Wildlifesydney.com.au.xml]: Mismatch:
calendar.wildlifesydney.com.au

**Comment by J0WI** [comment]
There are a few mismatches that needs to be documented.
Rules lgtm.

**Comment by J0WI** [comment]
@youdly are you still working on this?

**Comment by youdly** [comment]
Added comment.

**Pull request merged**